### ISSUE TITLE
Cuckoo API not selecting attachments of a .msg file by default

### ISSUE THREAD

**Issue by sabyasachisamanta** [cuckoosandbox/cuckoo#2715]
Thanks for creating an issue! But first: did you read our community guidelines?
https://cuckoo.sh/docs/introduction/community.html

**Comment by sabyasachisamanta** [comment]
Yes i have read it and could not find anything related to it. Would really help if some can let me on how to do it for a curl request.

**Comment by mkrsfcmp** [comment]
As far as I can see there is no analysis package to handle Outlook msgs, so you have to either:
1) Create your own analysis package to handle Outlook msg (probably opening Outlook inside the guest VM and so on - can be tricky), or
2) Modify your script to extract attachments first and then submitting only attachments to cuckoo and let it run the chosen package apropriate for given file.

**Comment by sabyasachisamanta** [comment]
Ok. Thanks!
